docs/design/DESIGN_HANDOFF_OUTPUT_v2.md

```
@@ -0,0 +1,272 @@
+
+## 2. Tokens nuevos
+
+**Ninguno.** Estas pantallas usan el sistema cerrado en v1.1.
+
+---
+
+## 3. Componentes
+
+### 3.1 Nuevos a crear
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)